Story/231913 navigation across the service should be easy with use of correct call to action buttons

PauliePension opened the pull request:
Removed chevron from primary and warning tpr buttons and *2 right padding.
deleted the redundant tpr-button--no-next-step class.

PauliePension committed:
#231913

Related Work Items: #231

PauliePension committed:
#23913 clean up changes

PauliePension committed:
#231913 removed the tpr-button--no-next-step class

PauliePension committed:
#231913 incremented patch leve

PauliePension committed:
Merge branch 'develop' into story/231913-Navigation-across-the-service-should-be-easy-with-use-of-correct-call-to-action-buttons

PauliePension committed:
Merge branch 'develop' into story/231913-Navigation-across-the-service-should-be-easy-with-use-of-correct-call-to-action-buttons

sussexrick commented on ThePensionsRegulator.Frontend/Styles/_tpr-button.scss:
```diff
@@ -38,106 +36,110 @@ govuk-button:focus:not(:active):not(:hover) {
+    background-color: $tpr-colour-white;
+    border-color: $tpr-colour-cornflower;
+}
+// Remove chevron where it's not wanted
```
Looks like this rule can be removed. The start button should still have a chevron - I guess this is removing a second chevron.

PauliePension replied:
done

sussexrick replied:
The following group of selectors (was Lines 83-87, now 74-79) are also removing the chevron where it's not wanted.

sussexrick commented on ThePensionsRegulator.Frontend/Styles/_tpr-button.scss:
```diff
@@ -18,17 +18,15 @@ $tpr-button-padding: 20px;
     color: $tpr-colour-eclipse;
 }
 
-.tpr-button--no-next-step, .govuk-file-upload::file-selector-button {
```
Removing this class is more than just deleting this line - if you search across the repo for it, you'll find it used in examples and components.

PauliePension replied:
done
deleted all through the app(s) and usync data

sussexrick commented on ThePensionsRegulator.Frontend/Styles/_tpr-button.scss:
```diff
@@ -1,10 +1,10 @@
+// tools like the govuk-colour() mixin that might apply more widely than buttons, rather than
 // defining a specific variable.
 //
 // The chevron image is from Material UI https://fonts.google.com/icons?selected=Material+Icons+Outlined:chevron_right
```
This comment is now misleading and can be removed.

PauliePension replied:
done

sussexrick commented on ThePensionsRegulator.Frontend/Styles/_tpr-button.scss:
```diff
@@ -18,17 +18,15 @@ $tpr-button-padding: 20px;
     line-height: govuk-px-to-rem(33);
     border-radius: 8px;
     box-shadow: none;
-    background: $govuk-button-background-colour url(/_content/ThePensionsRegulator.Frontend/tpr/chevron_right_white_24dp.svg) right center no-repeat;
```
You can delete the file too, unless it's the same one used on the start now button.

PauliePension committed:
comment removed
image deleted
deleted "tpr-button--no-next-step" class from content, examples and usync data.
removed 'chevron not wanted' class

PauliePension committed:
#231913 bumped patch value

sussexrick reviewed (changes requested):
In both example apps we have the buttons page refers to the difference created by the "tpr-button--no-next-step" class, which is now obsolete. The introductory paragraph and second set of examples can go, in both apps.

<img width="1547" height="836" alt="Two sets of identical buttons" src="https://github.com/user-attachments/assets/0b173df4-a1f5-40aa-b8fb-21eb51e5393b" />

PauliePension committed:
Commented out another class.
Updated sample apps view / umbraco

sussexrick commented on CheapNuget.ps1:
```diff
@@ -0,0 +1,20 @@
+
+$ValidPath = Test-Path -Path .\tempnuget
```
I don't think you meant to include this file...

PauliePension committed:
deleted developer file from branch
